ichnaea/taskapp/config.py

```diff
@@ -102,11 +102,10 @@ def init_worker(
 
     celery_app.geoip_db = configure_geoip(raven_client=raven_client, _client=_geoip_db)
 
-    # configure data queues
-    celery_app.all_queues = all_queues = set([q.name for q in TASK_QUEUES])
-
+    # configure data queues and build set of all queues
+    all_queues = set([q.name for q in TASK_QUEUES])
     celery_app.data_queues = data_queues = configure_data(redis_client)
-    all_queues = all_queues.union(
+    celery_app.all_queues = all_queues.union(
```

Comment:
This is the money fix. Previously, `all_queues` would end up with a set of all the queues: task and data and export. However, since that wasn't persisted anywhere, it'd get lost. `celery.all_queues` was only the task queues.

If we do this same fix in the 1.5 branch, we'll be able to see metrics on the data queues. I'll do that in a followup PR.